src/react-native/javascript.ts

```diff
@@ -127,15 +156,32 @@ ${
   // Configure Session Replay
   replaysSessionSampleRate: ${sessionReplaySampleRate},
   replaysOnErrorSampleRate: ${sessionReplayOnErrorSampleRate},
-  integrations: [Sentry.mobileReplayIntegration()],
 `
     : ''
-}
+}${getSentryIntegrationsPlainTextSnippet(
+    enableSessionReplay,
+    enableFeedbackWidget,
+  )}
   // uncomment the line below to enable Spotlight (https://spotlightjs.com)
   // spotlight: __DEV__,
 });`;
 }
 
+export function getSentryIntegrationsPlainTextSnippet(
+  enableSessionReplay = false,
+  enableFeedbackWidget = false,
+) {
+  if (!enableSessionReplay && !enableFeedbackWidget) {
+    return '';
+  }
+  return `  integrations: [${
+    enableSessionReplay ? 'Sentry.mobileReplayIntegration()' : ''
+  }${enableSessionReplay && enableFeedbackWidget ? ', ' : ''}${
+    enableFeedbackWidget ? 'Sentry.feedbackIntegration()' : ''
```

Comment:
Just adding the integration for now. We may add more options with with feature drop 2 (e.g. enable capture screenshot)